code-challenges.js

```diff
@@ -1,30 +1,136 @@
+//                 console.log(smallArr[j]) //ISSUE: only pushes "apple"
+//                 return newArray.push(smallArr[j])
+//             //return that index and push it into the empty array, newArray 
+//             } else {
+//             }
+//         }
+//     return newArray
+// }
+// console.log(arrayOfWords);
+// console.log(aLet(arrayOfWords));
 
+//------------------first attempt was a fail, second attemp succeeded---------------------------
 
-
+const aLet = (array) => {
+    smallArr=[]
+        for (var i=0; i < array.length; i++) {
+            smallArr.push(array[i].toLowerCase())
+        }
+    //smallArr=["apple", "banana", "plum", "cherry", "kiwi", "peach"] --> works
+        newArray=smallArr.filter (value => {
+            return value=value.includes('a')
+        })
+        return newArray
+    }
+    console.log(arrayOfWords);
+    console.log(aLet(arrayOfWords))
 
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
-
+//           0  1  2  3  4
 var hand1 = [5, 5, 5, 3, 3]
 // Expected output: true
 var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
+var hand4 = [5,3,5,5,3]
+
+// pseudocode:  Must add sort method to array
+//              1. create a function that iterates through array and checks if three of the coinciding values are the same, thanks to the sort method
+//              2. push those values into an empty array, "fullHouseArr"
+//              3. iterate through the remaining values to determine if they match
+//              4. if they match, push the remaining values into the array holding the three matching values (fullHouseArr)
+//              5. if the fullHouseArr.length = 4, console.log "You have a full house"
+
+const letsSee = (array) => {
+let sortedArr = array.sort()
+//             0  1  2  3  4
+//var hand1 = [3, 3, 5, 5, 5]
+fullHouseArr=[]
+remaining=[]
+    for (let i = 0; i < sortedArr.length; i++) {
+        if (sortedArr[i + 2] == sortedArr[i+1] 
+            && sortedArr[i+1] == sortedArr[i]
+            && sortedArr[i + 2] == sortedArr[i]) {
+            fullHouseArr.push(sortedArr[i+2])
+            fullHouseArr.push(sortedArr[i+1])
+            fullHouseArr.push(sortedArr[i])
+        } else {
+            remaining.push(sortedArr[i])
+        }
+    }
+    for (let k = 0; k < remaining.length-2; k++) {
+        if (remaining[k + 1] == remaining[k]) {
+            fullHouseArr.push(remaining[k])
+            fullHouseArr.push(remaining[k+1])
+        }
+    }
+    if (fullHouseArr.length==5){
+        return `Congrats, your hand of "${fullHouseArr}" is a full house!`
+    } else {
+        return "Womp, you do not have a full house"
+    }
+}
+console.log(letsSee(hand1))
+console.log(letsSee(hand2))
+console.log(letsSee(hand3))
+console.log(letsSee(hand4))
```

Comment:
YOU DID IT! Nice work!!! 

My only advice I'd give here is to be careful of adding extra flair returns. While I love the flair of your outputs, the requirements/expected output is only expecting a true or a false. When we're working with projects, we need to keep this in mind as the expected output might be required to be used in another function/program elsewhere in our application.